resources/views/welcome.html

```diff
@@ -40,6 +40,7 @@
           <a class="dropdown-item" id="interval_all">
             All
           </a>
+          <a href="/select" class="dropdown-item">Search Hardware</a>
```

Comment:
Edit the `href` using the [`route` helper](https://laravel.com/docs/11.x/routing#generating-urls-to-named-routes) and the name of the route instead of the url